tick/inference/robust.py

```diff
@@ -0,0 +1,50 @@
+# License: BSD 3 clause
+
+import numpy as np
+
+
+def std_mad(x):
+    """
+    Robust estimation of the standard deviation, based on the Corrected Median
+    Absolute Deviation (MAD) of x.
+    This computes the MAD of x, and applies the Gaussian distribution
+    correction, making it a consistent estimator of the standard-deviation
+    (when the sample looks Gaussian with outliers).
+
+    Parameters
+    ----------
+    x : `numpy.array`
```

Comment:
`numpy.array` is not linked in the doc (at least in local), while `np.ndarray` is